### ISSUE TITLE
AHS submission dry run or task validation. 

### ISSUE THREAD

**Issue by weinbe58** [aws/amazon-braket-sdk-python#550]
**Describe the feature you'd like**
I want the ability to perform a dry run for AHS task submission. The idea would be to validate the task without actually submitting the task to the queue. 

**How would this feature be used? Please describe.**
This would be useful to prototype tasks and help when submitting batches of tasks: validating all tasks before submitting them to the queue.

**Describe alternatives you've considered**
Not really any clear option to do this.

**Comment by christianbmadsen** [comment]
@weinbe58 Thank you for opening this request. The ask makes sense. Could you add a bit of context to your specific use case? Is it related to speed of debugging? Ability to build in exception handling during larger workloads? Something else? Thanks!

**Comment by weinbe58** [comment]
> Is it related to speed of debugging? 

Yes, one of the most common issues is trying to fit the minimum time step.  

> Ability to build in exception handling during larger workloads? 

yes, many jobs require running batches of tasks that are related a bunch of related tasks and those tasks are generated programmatically where some errors can happen when generating some of those tasks.

**Comment by weinbe58** [comment]
@christianbmadsen Any update on this issue?

**Comment by christianbmadsen** [comment]
@weinbe58 your ask makes sense. We dont have a timeline for implementing dry-run internally. Would you be interested in picking up such a contribution if we open up an issue with implementation guidelines for you and help you with the PR reviews?
